Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/util/wrappers/LayerMenuViewInteraction.java

```diff
@@ -106,4 +118,34 @@ public LayerMenuViewInteraction performDeleteLayer() {
 				.perform(click());
 		return this;
 	}
+
+	public LayerMenuViewInteraction performHideLayer(int position) {
```

Comment:
Hide and Unhide Layer have the same code, if it is working as intended merge it into one function

Reply:
Stimmt, hab es so implementiert, dass es übersichtlicher ist, werde ich zusammenmergen und "toggleLayerVisibility" nennen 👍